### PR TITLE
Enhance/dank memes

### PR DESCRIPTION
**Pull request by anapp4that**
Added a feature to display the number of dank memes huskie bot has
Added a print out that tells you the dank memes number of the image you uploaded when doing a single dank image upload
Indexed images at one so it makes more sense about how many images exist, and that theres not an "extra" image at 0

### CODE REVIEW

File: cogs/dank_memes.py [hunk 1]
```diff
@@ -142,18 +143,29 @@ class DankMemes(BaseCog):
             Dank Image
 
         """
+        if ctx.invoked_subcommand is None:
+            images = sorted(os.listdir(MEDIA_PATH))
+            if len(images) == 0:
+                await ctx.send('I don\'t have any images to shitpost with')
```
Comment (willpoorman) [inline cogs/dank_memes.py]: I'd suggest switching this to double quotes to not have to escape the apostrophe

Comment (anapp4that) [reply cogs/dank_memes.py]:
```suggestion
                await ctx.send('I don\'t have any images to shitpost with')
```

Can do, This was how @michaelhillebrand had it and I didn't touch that line

Comment (willpoorman) [reply cogs/dank_memes.py]: Gotcha, seems like a good time to switch it in my opinion, since we're already editing this section

Comment (anapp4that) [reply cogs/dank_memes.py]: Done

File: cogs/dank_memes.py [hunk 2]
```diff
@@ -116,18 +116,19 @@ class DankMemes(BaseCog):
         if not ctx.message.attachments:
             await ctx.send('No file detected')
         else:
+            images = sorted(os.listdir(MEDIA_PATH))
```
Comment (michaelhillebrand) [inline cogs/dank_memes.py]: remove sorted() wrapper

File: cogs/dank_memes.py [hunk 3]
```diff
@@ -116,18 +116,19 @@ class DankMemes(BaseCog):
...
             async with ctx.typing():
                 for attachment in ctx.message.attachments:
                     file = await download(attachment.url)
                     await self._process_image(file)
-                    await ctx.send('Dank image uploaded successfully')
+                    await ctx.send(f'Dank image {len(images) + 1} uploaded successfully')
```
Comment (michaelhillebrand) [inline cogs/dank_memes.py]: Maybe rephrase this to something like "dank image uploaded successfully at index {index}"

File: cogs/dank_memes.py [hunk 4]
```diff
@@ -142,18 +143,29 @@ class DankMemes(BaseCog):
...
+                        await ctx.send(index + 1, file=discord.File(os.path.join(MEDIA_PATH, images[index])))
+                    else:
+                        raise RuntimeError
+                except (ValueError, RuntimeError, IndexError):
+                    await ctx.send('That is not a valid meme')
+
+    @dank.command(name='count')
+    async def dank_count(self, ctx):
+        """HuskieBot will say how many memes it currently has"""
         images = sorted(os.listdir(MEDIA_PATH))
```
Comment (michaelhillebrand) [inline cogs/dank_memes.py]: remove sorted() wrapper
